### PR TITLE
Improve SafeMath test coverage.

### LINKED ISSUE #1563: Improve SafeMath tests
The SafeMath tests need some love. We should add a helper to test all operations behave the same way when their arguments are swapped. And we should make sure to be testing all edge cases.

### PR DESCRIPTION
**Pull request by nventuro**
Fixes #1563.

The issue also mentions missing edge cases, but I didn't find any, nor are they listed there.

### CODE REVIEW

File: test/drafts/SignedSafeMath.test.js [hunk 1]
```diff
@@ -8,34 +8,43 @@ contract('SignedSafeMath', function () {
     this.safeMath = await SignedSafeMathMock.new();
   });
 
+  async function testCommutative (fn, lhs, rhs, expected) {
```
[the remark this answers is not in the document]
Comment (nventuro) [reply test/drafts/SignedSafeMath.test.js]: `function` is sadly a reserved keyword in JavaScript.
